dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/consumer/TaskPriorityQueueConsumer.java

```diff
@@ -72,6 +75,12 @@ public class TaskPriorityQueueConsumer extends BaseDaemonThread {
     @Autowired
     private TaskPriorityQueue<TaskPriority> taskPriorityQueue;
 
+    /**
+     * task failed queue
+     */
+    @Autowired
+    private TaskPriorityQueue<TaskPriority> taskPriorityDispatchFailedQueue;
```

Comment:
This bean is the same with `taskPriorityQueue`.

Reply:
Yes, I will add the implementation of the failure queue under queue.